From 3245e95a5cae026fc2fd255dabcee6ff12d71897 Mon Sep 17 00:00:00 2001
From: GuaiYiHu <guaiyihu@foxmail.com>
Date: Thu, 13 Aug 2020 12:25:09 +0800
Subject: [PATCH 028/115] drivers: gpu: drm: msm: sde: Increase sde stage to
 map zpos changes

 * Another stage should be added, otherwise fingerprint payment won't work in some apps
 like Alipay and Jingdong.

Change-Id: Ibbd928b6d75c17f9c6c2e8b6f72b2d17aa48eb94
---
 drivers/gpu/drm/msm/sde/sde_hw_mdss.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/msm/sde/sde_hw_mdss.h b/drivers/gpu/drm/msm/sde/sde_hw_mdss.h
index bd3ef45df..fbbec709a 100644
--- a/drivers/gpu/drm/msm/sde/sde_hw_mdss.h
+++ b/drivers/gpu/drm/msm/sde/sde_hw_mdss.h
@@ -174,6 +174,7 @@ enum sde_stage {
 	SDE_STAGE_8,
 	SDE_STAGE_9,
 	SDE_STAGE_10,
+	SDE_STAGE_11,
 	SDE_STAGE_MAX
 };
 enum sde_dspp {
-- 
2.52.0

